rxdogtag/src/main/java/com/uber/rxdogtag/DogTagCompletableObserver.java

```diff
@@ -55,7 +55,11 @@ public void onSubscribe(Disposable d) {
 
   @Override
   public void onError(Throwable e) {
-    reportError(config, t, e, null);
+    if (delegate instanceof TryOnError) {
+      guardedDelegateCall(e2 -> reportError(config, t, e2, "onError"), () -> delegate.onError(e));
```

Comment:
I think we should just use the existing `guardObserverCallbacks` check for onError as well.